excel/src/main/java/org/apache/metamodel/excel/ExcelConfiguration.java

```diff
@@ -38,25 +38,38 @@
 	public static final int NO_COLUMN_NAME_LINE = 0;
 	public static final int DEFAULT_COLUMN_NAME_LINE = 1;
 
+	private final int getNumberOfLinesToScan;
 	private final int columnNameLineNumber;
 	private final ColumnNamingStrategy columnNamingStrategy;
 	private final boolean skipEmptyLines;
 	private final boolean skipEmptyColumns;
+	private final boolean detectColumnTypes;
 
 	public ExcelConfiguration() {
 		this(DEFAULT_COLUMN_NAME_LINE, true, false);
 	}
 
     public ExcelConfiguration(int columnNameLineNumber, boolean skipEmptyLines, boolean skipEmptyColumns) {
-        this(columnNameLineNumber, null, skipEmptyLines, skipEmptyColumns);
+        this(columnNameLineNumber, null, skipEmptyLines, skipEmptyColumns, false, 1000);
```

Comment:
It would be nice if the default value of a "1000" was in a constant, and maybe you want to set the default to zero in case you pass "false" as an argument for the `detectColumnTypes` property.

Reply:
Done with next commit.